### PR TITLE
chore(na): update hosting requirements and considerations per feedback

### PR DESCRIPTION
**Pull request by mrjones-plip**
# Description

update hosting requirements and considerations per feedback

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by mrjones-plip** [comment]
cc @esthermmoturi

File: content/en/hosting/_index.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ Before beginning any of these guides, be sure to meet all of the [CHT hosting re
...
+* **Alerting** - How will alerts be sent in the case of downtime or degraded service?  While [Watchdog]({{< relref "hosting/monitoring/introduction" >}}) can be set up to monitor CHT Core instances - which monitoring system will be used to alert on OS level warnings?
+* **Power failures and unplanned restarts** - Will the server cleanly restart such that the CHT resumes service correctly?
+* **Backups** - What happens to the CHT data if there's a hard drive failure? Are there provisions for a [3-2-1 backup strategy](https://en.wikipedia.org/wiki/Backup#Storage)?
+* **Disaster Recovery** - What happens if there is a flood at the facility and on-site active and backup data are destroyed?
+* **Scale** - What happens when the hardware deployed needs to be upgraded to increase capacity?
+* **Updates** - By definition TLS certificates expire and software needs to be updated - how will the deployment get these updates on a regular basis? CHT Core updates happen many times through out the year - do you have a maintenance schedule for these and staff who are trained to do the upgrade?
```
Comment (garethbowen) [inline content/en/hosting/_index.md:31]: For me, TLS renewals are separate from Updates. What about this?

```suggestion
* **Updates** - CHT Core updates happen many times throughout the year - do you have a maintenance schedule for these and staff who are trained to do the upgrade?
* **Renewals** - Services such as TLS certificates and domains name registration expire periodically and will make the server inaccessible - do you have a schedule for monitoring and renewing these in a timely manner?
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Thanks for running with this! Suggestions inline.

File: content/en/hosting/_index.md [hunk 2]
```diff
@@ -23,11 +23,11 @@ Before beginning any of these guides, be sure to meet all of the [CHT hosting re
...
-* Scale - What happens when the hardware deployed needs to be upgraded to increase capacity?
-* Updates - By definition TLS certificates expire and software needs to be updated - how will the deployment get these updates on a regular basis?
-* Security - While the TLS certificate will protect data on the LAN, is the server hard drive encrypted in the event of property theft?
-* Privacy - The CHT inherently carries sensitive patient medical information in the database. Are there sufficient measures in place to protect this sensitive data?
+* **Alerting** - How will alerts be sent in the case of downtime or degraded service?  While [Watchdog]({{< relref "hosting/monitoring/introduction" >}}) can be set up to monitor CHT Core instances - which monitoring system will be used to alert on OS level warnings?
+* **Power failures and unplanned restarts** - Will the server cleanly restart such that the CHT resumes service correctly?
+* **Backups** - What happens to the CHT data if there's a hard drive failure? Are there provisions for a [3-2-1 backup strategy](https://en.wikipedia.org/wiki/Backup#Storage)?
```
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply content/en/hosting/_index.md]: good thinking! fixed in d57c6553

**Comment by mrjones-plip** [comment]
@garethbowen -  if there's no more changes requested, please merge so we can get these changes live as I'll be away from keyboard for a bit!

I'll otherwise tend to further improvements upon my return.

**Review by garethbowen — APPROVED** [review]
Definite improvement, thanks!

**Pull request merged**